src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -917,6 +1103,41 @@ public static void insertStanzaInAppServerXML(RemoteRobot remoteRobot, String st
         }
     }
 
+    /**
+     * Deletes a string of text from the currently focused editor
+     *
+     * @param remoteRobot      The RemoteRobot instance.
+     * @param textToDelete     The string to delete
+     */
+    public static void selectAndDeleteTextInJavaPart(RemoteRobot remoteRobot, String textToDelete){
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(30));
+        clickOnFileTab(remoteRobot, "server.xml");
```

Comment:
how come we are always clicking on `server.xml` here? Should the file to delete text from be a parameter for this method?